lib/ar2dto/dto.rb

```diff
@@ -4,13 +4,30 @@ module AR2DTO
   class DTO
     class << self
       def [](original_model)
-        Class.new(self) do |_klass|
-          attr_accessor(*original_model.attribute_names)
+        Class.new(self) do
+          attr_reader(*original_model.attribute_names)
+
+          private
+
+          attr_writer(*original_model.attribute_names)
+
+          define_singleton_method :original_model do
+            original_model
+          end
```

Comment:
Just curious, what does this do?

Reply:
this is defining a class method in the dynamic class called original_model, that returns the class name of the model (e.g. User) so I can use it in other places like in `lib/ar2dto/active_model.rb`